content/blog/2025/community-ownership/index.md

```diff
@@ -0,0 +1,74 @@
+
+1. A company releases software under an open source license and professes to build an open source community around it.
+2. However, they retain the rights to all of the code in their projects through a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) (CLA). This generally means that contributors must _give up the rights to their contribution_ in order to make that contribution.
+3. Once their product has gained traction and it is in their interests, the company can _change the license_ to whatever they wish (even one that is not open source) because they retain the rights to all contributions in the codebase.
+4. They then leverage this new position as owners of a proprietary project to extract business value or grow their position in a market.
+
+Think this sounds unlikely? Here are just a few recent examples of companies that have switched their license after many years of releasing their technology under an open source license:
+
+- [Redis](https://redis.io/blog/redis-adopts-dual-source-available-licensing/)
+- [Hashicorp / Terraform](https://www.hashicorp.com/blog/hashicorp-adopts-business-source-license)
+- [Elastic Search](https://en.wikipedia.org/wiki/Elasticsearch#Licensing_changes)
+
+We want to ensure our commities that 2i2c is not headed down this path, in order to give them confidence in treating us as a long-term service partner.
+
+## What does this change about 2i2c's open source commitment?
+
+In short: nothing. These are already the principles that 2i2c was committed to from its inception. However, we wanted to make these commitments more formally in order to give ourselves more accountability to sticking with them, and to provide more transparency for our community members and stakeholders.
+
+## Who is this for?
+
+We imagine three audiences for this policy:
+
+1. **2i2c present and future staff** who want to ensure that their organization remains committed to our open principles. This document provides a sense of psychological safety to have bold discussions about structuring our approach to open source.
+2. **Member communities and 2i2c stakeholders** who _need_ to have a better understanding of the guarantees that we provide in order to trust 2i2c as a service developer and provider. This is similar to the effect our [Right to Replicate](/right-to-replicate) has.
+
+## We'd love feedback
+
+We hope that these ideas both clarify our intent and the reason that we think it's important. We'd love feedback about early refinements to these principles in order to make them more effective, as well as ways that we can provide more community oversight and participation in evolving these policies moving forward. If you have any thoughts to share, please send us feedback to `[hello@2i2c.org](mailto:hello@2i2c.org)`.
```

Comment:
I thought we discontinued this email. Where does this go to?

Reply:
ok I'll switch this to ask for an issue in the team compass. I don't know what the recommended way for others to provide feedback would be

Reply:
@choldgraf I just tested where this goes. i'm not sure but i do know at least it lands on my inbox :D so we can let this be for now - better than team-compass. let's not tweak too much - i just wanted to make sure the email goes somewhere